### PR TITLE
Inside Rust: announce Pietro joining the core team

### PR DESCRIPTION
**Pull request by nrc**
I've added a short note about Aidan at the bottom, but we shouldn't post this until we've checked that is OK with him.

cc @rust-lang/core

### CODE REVIEW

File: posts/inside-rust/2020-02-27-pietro-joins-core-team.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: "Pietro Albini has joined the core team"
+author: Nick Cameron
+team: Core Team
+---
+
+We are very happy to announce that Pietro Albini @pietroalbini has joined the core team. Pietro joined us back on December 24th 2019 (a Christmas present for the core team!), but we have been a bit late in announcing it (sorry Pietro!).
```
Comment (pietroalbini) [inline posts/inside-rust/2020-02-27-pietro-joins-core-team.md]:
```suggestion.
We are very happy to announce that [Pietro Albini](https://github.com/pietroalbini) has joined the core team. Pietro joined us back on December 24th 2019 (a Christmas present for the core team!), but we have been a bit late in announcing it (sorry Pietro!).
```

File: posts/inside-rust/2020-02-27-pietro-joins-core-team.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+Pietro is the co-leader of the infrastructure team, and a member of the crates.io, docs.rs, and release teams, and the security response and Rustfix working groups. He has been instrumental in setting up and maintaining the various services and technology which keeps the Rust project moving quickly and smoothly.
+
+Pietro will continue to focus on infrastructure issues, starting with setting up a working group to coordinate Rust's web presence - the website, crates.io, docs.rs, etc.
```
Comment (pietroalbini) [inline posts/inside-rust/2020-02-27-pietro-joins-core-team.md]: I think the goal was to create a team, not a WG?

File: posts/inside-rust/2020-02-27-pietro-joins-core-team.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: "Pietro Albini has joined the core team"
+author: Nick Cameron
+team: Core Team
+---
+
+We are very happy to announce that Pietro Albini @pietroalbini has joined the core team. Pietro joined us back on December 24th 2019 (a Christmas present for the core team!), but we have been a bit late in announcing it (sorry Pietro!).
+
+Pietro is the co-leader of the infrastructure team, and a member of the crates.io, docs.rs, and release teams, and the security response and Rustfix working groups. He has been instrumental in setting up and maintaining the various services and technology which keeps the Rust project moving quickly and smoothly.
```
Comment (pietroalbini) [inline posts/inside-rust/2020-02-27-pietro-joins-core-team.md]: I'd remove the mention of rustfix, I didn't really do much there. Also I guess I'm also co-leading docs.rs now.

**Comment by pietroalbini** [comment]
Also, CI is failing.

**Comment by nikomatsakis** [comment]
LGTM, presuming @aidanhs approves

File: posts/inside-rust/2020-02-27-pietro-joins-core-team.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: "Pietro Albini has joined the core team"
+author: Nick Cameron
+team: the core team <https://www.rust-lang.org/governance/teams/core>
+---
+
+We are very happy to announce that [Pietro Albini](https://github.com/pietroalbini) has joined the core team. Pietro joined us back on December 24th 2019 (a Christmas present for the core team!), but we have been a bit late in announcing it (sorry Pietro!).
+
+Pietro co-leads of the infrastructure and docs.rs teams, and is a member of the crates.io and release teams, and the security response working groups. He has been instrumental in setting up and maintaining the various services and technology which keeps the Rust project moving quickly and smoothly.
```
Comment (pietroalbini) [inline posts/inside-rust/2020-02-27-pietro-joins-core-team.md]:
```suggestion
Pietro co-leads of the infrastructure and docs.rs teams, and is a member of the crates.io and release teams, and the security response working group. He has been instrumental in setting up and maintaining the various services and technology which keeps the Rust project moving quickly and smoothly.
```

**Comment by aidanhs** [comment]
r+

**Pull request merged**